addons/third-party-button/userscript.js

```diff
@@ -0,0 +1,26 @@
+﻿export default async function ({ addon, global, console }) {
+  addon.settings.addEventListener("change", () => console.log("changed!"));
+  const viewon = document.createElement("span");
+
+  viewon.innerHTML = "View on ";
+  viewon.title = "Third-party sites";
+  viewon.style = "font-size:15px;float:right;";
+  console.log(window.location.href);
+  var projectid = window.location.pathname.split("/")[2];
+  console.log(projectid);
+
+  if (addon.settings.get("forkphorus")) {
+    viewon.innerHTML += '<a title="forkphorus" href="https://forkphorus.github.io/#' + projectid + '">forkphorus</a>';
```

Comment:
Please do not use innerHTML, as it may create security risks.

Reply:
Huh?! Okay, what else should I do?

Reply:
`textContent`, but I'm not sure if it's possible to insert elements with `textContent`. If not, we can take the scenic route.

Reply:
Yeah I already thought about the fact that you *could* have a URL containing `"><script>/*evil*/</script><"` or something but since you can't enter one and since it would also be kinda stupid, it's not really a risk.

Reply:
Yeah, it is valid if it's done in a XSS, but, man, this is far from one.